examples/common/java/direct/worker/src/main/resources/wallet/wallet1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "address": "1a69b2eb604db8eba185df03ea4f5288dcbbd248",
+  "id": "9428da60-eafb-4771-a64b-fac009946bba",
+  "version": 3,
+  "Crypto": {
+    "cipher": "aes-128-ctr",
```

Comment:
You might want to change it to aes-256-gcm which is supported by TCF

Reply:
This wallet is just for testing, we can update to test different ciphers in the future.